backend/tests/v5/unit/schemas/tickets/templates.test.js

```diff
@@ -885,6 +885,7 @@ const testValidate = () => {
 			config: {
```

Comment:
If the validity test is moved in config schema I'd create another set of tests ni Validate method to ensure the tabular objest is valid